one-pager-maker/.eslintrc.cjs

```diff
@@ -31,6 +31,7 @@ module.exports = {
         "react"
     ],
     "rules": {
+        "react/prop-types": "off",        
```

Comment:
https://cpoint-lab.co.jp/article/202107/20531/

Reply:
You might not need to add this.
After you see my review to `UserItem.tsx` and fix it, please try to remove this and check wheather it works.

Reply:
Thank you! I was able to erase it